azext_iot/central/_help.py

```diff
@@ -477,12 +480,59 @@ def _load_central_device_groups_help():
     """
 
 
+def _load_central_file_upload_configuration_help():
+    helps[
+        "iot central file-upload-configuration"
```

Comment:
Can this command group be shortened to [az] `iot central file-upload-config` ?

Reply:
sure